docs/dev-guide/contrib-guides-taskfiles.md

```diff
@@ -0,0 +1,161 @@
+# Taskfiles
```

Comment:
Although not Taskfile-specific, do we want to mention in YAML how we place array elements into separate lines where they can't fit into one?

Reply:
Good point. I would suggest we do it in another PR when we add YAML guidelines, and then add a link to them from the Taskfile guidelines.